build_tools/github/download_data.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -ef
+
+DOWNLOAD_DATA=$(cat <<END
+from nilearn import datasets
+
+# Download all datasets used in examples
+# Specifying parameters such as n_subjects if amount of data
+# downloaded depends on them
+datasets.fetch_neurovault_motor_task()
+datasets.fetch_atlas_smith_2009()
+datasets.fetch_haxby()
+datasets.fetch_haxby(subjects=[], fetch_stimuli=True)
+datasets.fetch_spm_auditory()
+datasets.fetch_surf_fsaverage()
+datasets.fetch_surf_fsaverage('fsaverage')
+datasets.fetch_atlas_surf_destrieux()
+datasets.fetch_atlas_destrieux_2009()
+datasets.fetch_atlas_harvard_oxford('cort-prob-2mm')
+datasets.fetch_atlas_juelich('prob-2mm')
+datasets.fetch_adhd(n_subjects=1)
+datasets.fetch_icbm152_2009()
+datasets.fetch_localizer_button_task()
+datasets.fetch_atlas_basc_multiscale_2015(version='sym')
+datasets.fetch_atlas_msdl()
+datasets.fetch_atlas_allen_2011()
+datasets.fetch_atlas_pauli_2017()
+datasets.fetch_atlas_difumo(dimension=64)
+datasets.fetch_surf_nki_enhanced(n_subjects=1)
+datasets.fetch_megatrawls_netmats()
+datasets.fetch_mixed_gambles()
+datasets.fetch_miyawaki2008()
+datasets.fetch_oasis_vbm()
+datasets.fetch_atlas_yeo_2011()
+datasets.fetch_development_fmri(n_subjects=60)
+datasets.fetch_fiac_first_level()
+datasets.fetch_localizer_first_level()
+# TODO: see timeout error that happens sometimes
+#datasets.fetch_spm_multimodal_fmri()
```

Comment:
Sometimes during the data download step, the call to download this dataset results in a connection timeout error. It seems to never happen when the data is downloaded as part of the sphinx build step but I haven't really investigated it. In any case, I can remove it for now, the data will still be downloaded and cached, but just not in this step. This can be fixed in a later PR.